### PR TITLE
[AL-3620] Add open it in browser option for email messages 

### PR DESCRIPTION
**Pull request by Sunilkumarr**
## Summary
This will add an option for opening in web view
 
## Testing
Checked on a device with email messages

### CODE REVIEW

File: Sources/Views/ALKFriendMessageCell.swift [hunk 1]
```diff
@@ -222,7 +231,9 @@ open class ALKFriendMessageCell: ALKMessageCell {
                 constant: Padding.TimeLabel.bottom)
             ])
 
-        messageView.addGestureRecognizer(tapGesture)
```
Comment (ShivamPokhriyal) [inline Sources/Views/ALKFriendMessageCell.swift]: Why tapGesture is removed here?

File: Sources/Views/ALKEmailView.swift [hunk 2]
```diff
@@ -72,3 +72,91 @@ class ALKEmailTopView: UIView {
...
+            static let height: CGFloat = 20.0
+            static let left: CGFloat = 3.0
+
+        }
+    }
+
+    var emailInfo: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 1
+        let attributedStringforNowLabel =  NSMutableAttributedString(string: "Have trouble viewing?", attributes: [
```
[the remark this answers is not in the document]
Comment (Sunilkumarr) [reply Sources/Views/ALKEmailView.swift]: Kern is required for spacing text.

File: Sources/Views/ALKEmailView.swift [hunk 3]
```diff
@@ -72,3 +72,91 @@ class ALKEmailTopView: UIView {
...
+    private func setupConstraints() {
+        self.backgroundColor = .clear
+        self.addViewsForAutolayout(views: [emailInfo,emailLinkLabel])
+        self.bringSubviewToFront(emailLinkLabel)
+
+        NSLayoutConstraint.activate ([
+            emailInfo.topAnchor.constraint(equalTo: topAnchor),
+            emailInfo.leadingAnchor.constraint(equalTo: leadingAnchor),
+            emailInfo.heightAnchor.constraint(equalToConstant: Padding.EmailInfo.height),
+            emailInfo.widthAnchor.constraint(equalToConstant: Padding.EmailInfo.width),
```
[the remark this answers is not in the document]
Comment (Sunilkumarr) [reply Sources/Views/ALKEmailView.swift]: works fine iPhone5s

File: Sources/Controllers/ALKEmailWebViewController.swift [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+//  ALKEmailWebViewController.swift
+//  ApplozicSwift
+//
+//  Created by Sunil on 12/07/19.
+//
+
+import Foundation
+import WebKit
+
+class ALKEmailWebViewController : UIViewController,WKNavigationDelegate {
```
[the remark this answers is not in the document]
Comment (Sunilkumarr) [reply Sources/Controllers/ALKEmailWebViewController.swift]: @ShivamPokhriyal check

**Pull request merged**